test/integration/targets/azure_rm_virtualmachine_b/aliases

```diff
@@ -0,0 +1,4 @@
+cloud/azure
```

Comment:
Definitely don't want to duplicate the test role; should just add a new variable like `use_plan` to the defaults in the original test role, and set up the plan dict or set it to `omit` if use_plan is false. Might also be worth exploring zikalino's idea about using unit tests

Reply:
Sorry not sure what is expected here.

We could do a unit test here if the account used for testing had authority to accept terms, and  check the returned object from `marketplace_agreements.create()` (as well as many other cases). Probably out of scope for this PR.

Reply:
The tests are broken into two due to timeouts BTW.

Reply:
Our test system won't understand the association between modules and tests with the test targets named this way. We shouldn't need to split the tests up like this. What is the timeout you're referring to?

Reply:
@mattclay: The CI seems to have a maximum time it allows a test to run after which the test `times out`. The `azure_rm_virtualmachine` tests take so long this will happen every time if we do all the tests at once.
 Can you look at http://docs.shippable.com/ci/custom-timeouts/ and see if you can adjust the timeout on this (or all) tests.

Otherwise what can we do? We can remove idempotency tests, but there are already way too few tests here as it is.

Reply:
@spmp What are the run times on the tests before and after your changes?

Reply:
@mattclay:
Tests run from my local machine as:
```
time ansible-test integration azure_rm_virtualmachine --tox --python 3.6 -v --allow-unstable --allow-destructive
```
The Azure resource `location` is not `eastus`, consequently one assert check is disabled.

* `ansible:devel`: 24 minutes
* `spmp:feature/os_disk_size_accept_terms`: 34 minutes

Reply:
I will push with tests combined - even tho this will fail - pending an increase in timeouts.